web3_proxy/src/app/mod.rs

```diff
@@ -366,15 +396,20 @@ pub struct Web3ProxyAppSpawn {
     pub background_handles: FuturesUnordered<AnyhowJoinHandle<()>>,
     /// config changes are sent here
     pub new_top_config_sender: watch::Sender<TopConfig>,
+    /// watch this to know when to start the app
+    pub consensus_connections_watcher: watch::Receiver<Option<Arc<ConsensusWeb3Rpcs>>>,
 }
 
 impl Web3ProxyApp {
     /// The main entrypoint.
     pub async fn spawn(
         top_config: TopConfig,
         num_workers: usize,
-        shutdown_receiver: broadcast::Receiver<()>,
+        shutdown_sender: broadcast::Sender<()>,
     ) -> anyhow::Result<Web3ProxyAppSpawn> {
+        let rpc_account_shutdown_recevier = shutdown_sender.subscribe();
```

Comment:
lets call this `stat_buffer_shutdown_receiver`. I plan on adding more stats that it will handle emitting